components/rabbitmq/scripts/creation_validation.py

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env python
-
-from os.path import join, dirname
-
-from cloudify import ctx
-
-ctx.download_resource(
-    join('components', 'utils.py'),
-    join(dirname(__file__), 'utils.py'))
-import utils  # NOQA
-
-runtime_props = ctx.instance.runtime_properties
-
-IMMUTABLE_PROPERTIES = [
```

Comment:
we had a good idea with this, check that rabbitmq creds didn't change, because then agents won't be able to connect, for an in-place upgrade

Reply:
Perhaps, but this was never used, right?

Reply:
well it was back in 3.4..; we'll probably have to do something like this again when we do the new "managed" in-place upgrade (if..)

Reply:
Yeah, maybe, but keeping legacy code for reference seems like a bad idea

Reply:
oh, i'm not suggesting we keep it, just commenting on it ;P